### PR TITLE
add position docs

### PR DESCRIPTION
**Pull request by danethurber**
add position docs

### CODE REVIEW

File: docs/src/pages/components/position.mdx [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+# Position
+
+<Intro>
+  The Position component is a utility meant to help one control position of
+  elements relative to other elements, for example, tooltips above a button.
+</Intro>
+
+
+Positioning with the component
```
Comment (jaketrent) [inline docs/src/pages/components/position.mdx:18]:
```suggestion

## Examples

#### Using the React component
```

File: docs/src/pages/components/position.mdx [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+  />
+))
+
+export default Example
+```
+
+## Javascript Example
+
+The most important bits of this utility are available in pure JavaScript. To
+use, import the `/js` part of the utility.
```
Comment (jaketrent) [inline docs/src/pages/components/position.mdx:222]:
```suggestion
The most important bits of this utility are available in pure JavaScript.
```

File: docs/src/pages/components/position.mdx [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+
+```javascript noRender startExpanded
+import {
+  above,
+  below,
```
Comment (jaketrent) [inline docs/src/pages/components/position.mdx:227]:
```suggestion
  above, aboveLeft, aboveRight,
  below, belowLeft, belowRight,
```

File: docs/src/pages/components/position.mdx [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+
+
+## Usage & types
+
+<Usage
+  install="npm install @pluralsight/ps-design-system-position"
+  import={`
+import {
+  Above,
+  Below,
```
Comment (jaketrent) [inline docs/src/pages/components/position.mdx:294]:
```suggestion
  Above, AboveLeft, AboveRight,
  Below, BelowLeft, BelowRight,
```

Or possibly just

```
{ Above, /* etc */ }
```

File: docs/src/pages/components/position.mdx [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+<Types.Table>
+  <Types.Prop
+    name="children"
+    type="React.ReactNode"
+    desc="target element something will be relative to"
+    required
+  />
+  <Types.Prop
+    name="inNode"
+    type="HTMLElement"
```
Comment (jaketrent) [inline docs/src/pages/components/position.mdx:311]: Great position on the types. You're great at that.

File: docs/src/pages/components/position.mdx [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+    icon={<PlaceholderIcon />}
+    title="Target"
+    ref={ref}
+  />
+))
+
+export default Example
+```
+
+## Portal override
```
Comment (jaketrent) [inline docs/src/pages/components/position.mdx:156]:
```suggestion
### Portal override
```

File: docs/src/pages/components/position.mdx [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+        Hover me
+      </Button>
+    </RightOf>
+  )
+}
+
+export default Example
+```
+
+## Positions
```
Comment (jaketrent) [inline docs/src/pages/components/position.mdx:57]: I'd move this outside (below) the examples section, and keep it an h2.

**Review by jaketrent — COMMENTED** [review]
Something weird going on with Positions section:
[positionScroll.mov.zip](https://github.com/pluralsight/design-system/files/5271534/positionScroll.mov.zip)

Just a couple other org tweaks.

**Comment by danethurber** [comment]
@jaketrent i'm not sure what to do about the scroll jumpiness. in the old docs we iframed the examples so the position stayed constant. in the new docs i can't really do that and maintain the example/preview system we have setup. should i get rid of the live previews?

**Comment by jaketrent** [comment]
@danethurber I'd probably do this example as a one-off.  Have an editor, no preview. Then do a custom component with iframes (or whatever works) as the coupled "preview".

File: docs/src/components/frame/frame.module.css [hunk 8]
```diff
@@ -45,3 +45,9 @@
     margin-right: 0;
   }
 }
+
+.example {
+  border-radius: 0.2em;
+  margin: var(--psLayoutSpacingMedium);
+  background: var(--psColorsWhite);
+}
```
Comment (danethurber) [inline docs/src/components/frame/frame.module.css:53]: iframes will always be in light theme. i could pass the prop through the message bus and handling theme toggling...but i don't want too 😭

File: docs/src/components/frame/frame.tsx [hunk 9]
```diff
@@ -53,6 +53,9 @@ export const Frame: React.FC<Props> = props => {
     [styles.light]: themeName === Theme.names.light
   })
 
+  const isExamplePage = window.location.pathname.includes('examples')
+  if (isExamplePage) return <ExamplePage>{children}</ExamplePage>
+
```
Comment (danethurber) [inline docs/src/components/frame/frame.tsx:58]: this feels so hacky...but i couldn't get other approaches to work

File: docs/src/content/components/position.mdx [hunk 10]
```diff
@@ -0,0 +1,236 @@
...
+```
+
+## Positions
+
+<IframeResizer
+  frameborder="0"
+  src="/examples/position"
+  style={{ width: '1px', minWidth: '100%'}}
+/>
+
```
Comment (danethurber) [inline docs/src/content/components/position.mdx:192]: iframing in the example

File: docs/src/components/frame/frame.module.css (hunk 8 above)
Comment (jaketrent) [reply docs/src/components/frame/frame.module.css:53]: Could read the cookie on the iframe side and use your own Theme.Provider

Comment (danethurber) [reply docs/src/components/frame/frame.module.css:53]: the library i'm using to do the resize also supports the onMessage. i'll spike on it this morning now that i'm fresh and not hating the iframe 😄

File: docs/src/components/frame/frame.tsx (hunk 9 above)
Comment (jaketrent) [reply docs/src/components/frame/frame.tsx:58]: Needs a canUseDOM() protection for SSR.

**Review by jaketrent — APPROVED** [review]
Looks good.  I have a branch from this branch you can merge first if you like the alternative approach.

**Pull request merged**